### PR TITLE
Adds a codeship step to sign the Windows installer

### PR DESCRIPTION
**Pull request by jesseconductor**
New branch without the re-written history

### CODE REVIEW

**Comment by jesseconductor** [comment]
The associated, signed release is https://github.com/AtomicConductor/conductor_client/releases/tag/v2.12.9

I haven't been able to test with conductor companion yet as I don't want to trump any current builds. Waiting on details on how to do it.

File: codeship-steps.yml [hunk 1]
```diff
@@ -42,6 +42,9 @@
         - service: el7
           name: Build EL-7 installer with desktop client
           command: /src/installers/rpm/build.sh 7 --with_client
+  - name: Certify
+    service: certify
+    command: /src/installers/certify.sh --with_client
```
Comment (flebel) [inline codeship-steps.yml]: You need to prefix it with `bash`.

File: installers/certify.sh [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    DESKTOP_CLIENT_DIR=/artifacts/build/dc/win64
+    cp -r  ${DESKTOP_CLIENT_DIR}/* ${BUILD_DIR}/Conductor
+    version=$( cat /artifacts/build/dc/current-version.txt | cut -d" " -f1 )
+    RELEASE_VERSION="v${version}"
+fi
+    
+echo ${WINDOWS_INSTALLER_CERTIFICATE} | base64 -d - > /tmp/authenticode-certificate.p12
+
+osslsigncode sign -pkcs12 /tmp/authenticode-certificate.p12 -pass ${WINDOWS_INSTALLER_CERTIFICATE_PWORD} -n "Conductor Client" -i "https://www.conductortech.com/" -in /artifacts/conductor-${RELEASE_VERSION}.unsigned.exe -out /artifacts/conductor-${RELEASE_VERSION}.exe
+
```
Comment (flebel) [inline installers/certify.sh]: Can you remove the stray line breaks?

Comment (jesseconductor) [reply installers/certify.sh]: Fixed

File: codeship-steps.yml (hunk 1 above)
Comment (jesseconductor) [reply codeship-steps.yml]: fixed

File: installers/certify_companion_binaries.sh [hunk 3]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```
Comment (flebel) [inline installers/certify_companion_binaries.sh]: Can you add ` -e` to exit on errror (similar to `set -e`.)

**Pull request merged**